tests/kubernetes_test.py

```diff
@@ -390,6 +390,29 @@ async def test_patch_service(self, kube: kubetest.client.TestClient) -> None:
         await svc.refresh()
         assert svc.obj.metadata.labels["testing.opsani.com"] == sentinel_value
 
+@pytest.mark.applymanifests("manifests", files=["fiber-http.yaml"])
+async def test_get_latest_pods(kube: kubetest.client.TestClient) -> None:
+    kube.wait_for_registered()
+    # Cache initially created replicaset
+    _, old_rset = kube.get_replicasets().popitem()
+
+    # Generate a new replicaset
+    kube_dep = kube.get_deployments()["fiber-http"]
+    kube_dep.obj.spec.template.spec.containers[0].resources.requests["memory"] = "128Mi"
+    kube_dep.api_client.patch_namespaced_deployment(kube_dep.name, kube_dep.namespace, kube_dep.obj)
+
+    servo_dep = await servo.connectors.kubernetes.Deployment.read("fiber-http", kube.namespace)
+    # Timing is a bit tricky on this one
+    await asyncio.sleep(1.5)
```

Comment:
Is there an event we can watch for or a value we can poll? Every time we blindly use sleep values like this the damned test winds up becoming brittle later on and I have to go back and hammer on it again

Reply:
I'm looking bottom's up from the code at:

* https://github.com/tomplus/kubernetes_asyncio/blob/master/kubernetes_asyncio/client/models/v1_replica_set.py
* https://github.com/tomplus/kubernetes_asyncio/blob/master/kubernetes_asyncio/client/models/v1_replica_set_spec.py
* https://github.com/tomplus/kubernetes_asyncio/blob/master/kubernetes_asyncio/client/models/v1_replica_set_status.py

Reply:
Sure thing: https://github.com/opsani/servox/blob/fred/sol-319-k8s-connector-get_latest_pods-uses/tests/kubernetes_test.py#L406-L409